package.json

```diff
@@ -12,12 +12,19 @@
   "author": "Lucas Dohmen <lucas@dohmen.io> (https://lucas.dohmen.io/)",
   "license": "Apache-2.0",
   "dependencies": {
+    "@babel/core": "^7.1.2",
+    "@babel/plugin-transform-react-jsx": "^7.0.0",
+    "@babel/preset-env": "^7.1.0",
     "commonmark": "^0.28.1",
-    "faucet-pipeline-core": "^1.0.0",
+    "complate-stream": "^0.16.4",
+    "faucet-pipeline-core": "^1.1.0",
```

Comment:
You don't actually need faucet-core v1.1.0, do you? (Though it looks like `page.js` might benefit from the newly liberated `resolvePath` function.)